integration_test/packages/run_numpy.py

```diff
@@ -62,6 +62,10 @@ def genfromtxt(self):
         np.genfromtxt(file_name, delimiter=",",
                       missing_values="", filling_values=-1)
 
+    def load(self):
```

Comment:
Could you add a very brief docstring here? (Like the other functions in this module)